podcast.rss

```diff
@@ -6968,7 +6968,7 @@
     <item>
       <title>592  സൂറഃ നാസിയാത്ത് ആമുഖം_a</title>
```

Comment:
Can we follow the same format in item title as we did for previous chapters?

` <title>590.സൂറഃ നാസിയാത്ത് പാരായണം Part 1</title>`
` <title>592  സൂറഃ നാസിയാത്ത് ആമുഖം_a</title>`